backend/main.py

```
@@ -183,6 +214,111 @@ def update_note(self, attendance_id, note):
+
+    def sync_members(self):
+        """JyoginHubから部員一覧を取得してmembersテーブルに保存"""
+        config = self.get_hub_config()
+        if not config["url"] or not config["api_key"]:
+            return {"status": "error", "message": "Hub設定が未登録です"}
+
+        try:
+            req = urllib.request.Request(
+                f"{config['url']}/api/hub/members",
+                headers={"Authorization": f"Bearer {config['api_key']}"},
+            )
```

Ruff (0.15.9)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)